docs/content/latest/develop/query/pg_stat_activity.md

```diff
@@ -0,0 +1,146 @@
+---
+title: View Live Queries
+headerTitle: View Live Queries
+linkTitle: View Live Queries
+description: Analyze and diagnose running queries with pg\_stat\_activity
+menu:
+  latest:
+    identifier: pg_stat_activity
+    parent: develop
+    weight: 566
+isTocNested: true
+showAsideToc: true
+---
+
+The `pg_stat_activity` view provides information on currently running tasks, so you can use it to troubleshoot problems by, for example, identifying long-running idle in transaction sessions or very long running queries. For each server process, `pg_stat_activity` can be used to discover the process ID, active user, running query, state, time the last query started, and more.
+
+To query `pg_stat_activity`, use the `SELECT` statement, as with any other table:
+
+```SQL
+SELECT * FROM pg_stat_activity;
+```
+
+The `pg_stat_activity` view displays one row per server process, showing information related to the current activity of that process.
+
+The fields and their values are described in the following table.
+
+Field | Type | Description
+--- | --- | ---
+datid | oid | The object identifier (OID) of the database to which the backend is connected.
+datname | name | The name of the database to which the backend is connected.
```

Comment:
Style preferences: markdown tables should be "outlined" with pipes. Also, be sure to specify the alignment for each column (my suggestion left-aligns them all).

There's no need to make the heading rows wide enough so the table actually formats as a table in your text editor, like some people do, but each row should be obviously a table row, from beginning to end.

```suggestion
| Field | Type | Description |
| :---- | :--- | :---------- |
| datid | oid | The object identifier (OID) of the database to which the backend is connected. |
| datname | name | The name of the database to which the backend is connected. |
```